### PR TITLE
bugfix: performance.now function should return MS instead of nano

### PR DESCRIPTION
**Pull request by Pruxis**

### CODE REVIEW

**Review by Pruxis — COMMENTED** [review]
Unsure if correct but I think this solves
https://github.com/Jarred-Sumner/bun/issues/413

File: src/bun.js/bindings/ZigGlobalObject.cpp [hunk 1]
```diff
@@ -1496,7 +1496,7 @@ JSC_DEFINE_HOST_FUNCTION(functionPerformanceNow, (JSGlobalObject * globalObject,
     auto* global = reinterpret_cast<GlobalObject*>(globalObject);
     // nanoseconds to seconds
     uint64_t time = Bun__readOriginTimer(global->bunVM());
-    double result = time / 1000000000.0;
+    double result = time / 1000000000000000.0;
     return JSValue::encode(jsNumber(time));
```
Comment (FinnRG) [inline src/bun.js/bindings/ZigGlobalObject.cpp]: You are still returning the time variable. Please calculate double result = time / 1000000.0; and change jsNumber(time) to jsNumber(result)

Comment (Pruxis) [reply src/bun.js/bindings/ZigGlobalObject.cpp]: I overlooked that, the added 0's are incorrect and should be time / 1_000_000.0 I suppose

**Comment by afrokick** [comment]
Maybe should add a test like `expect(Math.abs(performance.now() - Date.now())).toBeCloseTo(0, 3);`?

https://github.com/Jarred-Sumner/bun/blob/main/test/bun.js/performance.test.js

**Pull request merged**